.pipelines/templates/template-push-images-to-acr.yml

```diff
@@ -10,4 +10,11 @@ steps:
 
     az acr login --name "$RP_IMAGE_ACR"
     make publish-image-aro
+    branch=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
Why not to put all this into makefile? now half of the logic is there, and other is here?

Reply:
mmmm, I could..